docs/warn-stealers.rst

```diff
@@ -14,4 +14,4 @@ Usage
 
 ::
 
-    warn-stealers [start|stop]
+    warn-stealers [enable|disable]
```

Comment:
Since it works with the enable API, could we document this as:
`enable warn-stealers`
?
I'm trying to standardize on "enable"/"disable" as the command instead of the argument (to make scripts and plugins look less distinct)

Reply:
I've also just been documenting the "enable" path (`enable warn-stealers`) instead of something like
`enable|disable warn-stealers` in order to make the syntax less confusing. Not sure if this is the right decision, but it's how all the other enableable tools are documented right now.

Reply:
the long term plan here with enableable tools is to collect them into an in-game configuration screen where they can be toggled. having them all use the `enable` syntax then will make much more sense.